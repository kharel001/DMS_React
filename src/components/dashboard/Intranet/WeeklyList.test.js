import React from 'react';
import WeeklyList from './WeeklyList';
import {
    configure,
    shallow,
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})

const dummy_data={
    data:[{id:1, data:'data1'},{id:2, data:'data2'}]
}
const setup = (props = {}) => {
    return shallow( < WeeklyList {...props}  />
    )
}


it("WeeklyList",()=>{
    const wrapper= setup(dummy_data);
    expect(wrapper).toMatchSnapshot();
})