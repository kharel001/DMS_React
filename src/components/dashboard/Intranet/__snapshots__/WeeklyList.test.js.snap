// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeeklyList 1`] = `ShallowWrapper {}`;
